api-tests/features/dl-auth-source-checks.feature

```diff
@@ -185,3 +228,33 @@ Feature: Authoritative source checks with driving licence CRI
       | given-names-and-address | kenneth-changed-given-name-driving-permit-valid  | coiAddress   |
       | family-name-only        | kenneth-changed-family-name-driving-permit-valid | coiNoAddress |
       | family-name-and-address | kenneth-changed-family-name-driving-permit-valid | coiAddress   |
+
+    @This
+  Scenario Outline: Change of details journey that attracts an invalid doc CI from auth source check
+    Given I activate the 'drivingLicenceAuthCheck' feature set
+    And the subject already has the following credentials
+      | CRI     | scenario                     |
+      | dcmaw   | kenneth-driving-permit-valid |
+      | address | kenneth-current              |
+      | fraud   | kenneth-score-2              |
+    When I start a new 'medium-confidence' journey
+    Then I get a 'page-ipv-reuse' page response
+    When I submit a 'update-details' event
+    Then I get a 'update-details' page response
+    When I submit a '<update-type>' event
+    Then I get a 'page-update-name' page response
+    When I submit a 'update-name' event
+    Then I get a 'dcmaw' CRI response
+    When I submit '<vc-scenario>' details to the CRI stub
+    Then I get a 'drivingLicence' CRI response
+    When I submit 'kenneth-driving-permit-needs-alternate-doc' details with attributes to the CRI stub
+      | Attribute | Values          |
+      | context   | "check_details" |
+    Then I get a 'sorry-could-not-confirm-details' page response
+
+    Examples:
+      | update-type             | vc-scenario                                      |
+      | given-names-only        | kenneth-changed-given-name-driving-permit-valid  |
+      | given-names-and-address | kenneth-changed-given-name-driving-permit-valid  |
+      | family-name-only        | kenneth-changed-family-name-driving-permit-valid |
+      | family-name-and-address | kenneth-changed-family-name-driving-permit-valid |
```

Comment:
I can't quite decide whether it's worth having all of these cases covered - feels like a lot of repetition but I suppose it is different routing for all of them...

I'm actually hoping that we might be able to get agreement to run the same COI check regardless of which name was indicated to be changed (i.e. allow _either_ name to change regardless of the original option selected), which could slim this down a bit.